app/javascript/readingList/components/ItemListLoadMoreButton.jsx

```diff
@@ -1,14 +1,14 @@
 // Load more button for item list
 import { h } from 'preact';
-import { PropTypes } from 'preact-compat';
+import PropTypes from 'prop-types';
 
 export const ItemListLoadMoreButton = ({ show, onClick }) => {
   if (!show) {
     return '';
   }
 
   return (
-    <div className="load-more-wrapper">
+    <div data-testid="load-more-button" className="load-more-wrapper">
```

Comment:
Do we need a `data-testid` on the div wrapper or would a `getByText('Load More')` be adequate enough.

Reply:
✅